indexer/feature_utils.cpp

```diff
@@ -438,4 +459,18 @@ string GetLocalizedFeeType(TypesHolder const & types)
   return localized_types[0];
 }
 
+string GetLocalizedSecondaryAtm(TypesHolder const & types)
+{
+  auto const & isAtmType = ftypes::IsATMChecker::Instance();
+  if (HasType(isAtmType, types) && !HasMainType(isAtmType, types))
```

Comment:
if (isAtmType(type) && ...)

And remove HasType function.

Reply:
Done